concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+Mountain-1978-267-35	35	varón	man	39	2106	MALE PERSON
+Mountain-1978-267-36	36	mujer	woman	40	962	WOMAN
+Mountain-1978-267-37	37	gente	people	41	789	PEOPLE
+Mountain-1978-267-38	38	marido	husband	42	1200	HUSBAND
+Mountain-1978-267-39	39	esposa	wife	43	1199	WIFE
+Mountain-1978-267-40	40	papá	father	44	1217	FATHER
+Mountain-1978-267-41	41	mamá	mother	45	1216	MOTHER
+Mountain-1978-267-42	42	agua	water	48	948	WATER
+Mountain-1978-267-43	43	río	river	49	666	RIVER
+Mountain-1978-267-44	44	isla	island	50	622	ISLAND
+Mountain-1978-267-45	45	lago	lake	51	624	LAKE
+Mountain-1978-267-46	46	pantano	swamp	52	1145	SWAMP
+Mountain-1978-267-47	47	fuente, ojo de agua	spring of water	53	849	SPRING (OF WATER)
+Mountain-1978-267-48	48	caída de agua	waterfall	54	943	WATERFALL
+Mountain-1978-267-49	49	catarata, raudal	rapids	55	1670	RAPIDS
+Mountain-1978-267-50	50	candela, fuego	fire	56	221	FIRE
+Mountain-1978-267-51	51	ceniza	ash	57	646	ASH
+Mountain-1978-267-52	52	carbón	charcoal	58	1188	CHARCOAL
+Mountain-1978-267-53	53	humo	smoke	59	778	SMOKE (EXHAUST)
+Mountain-1978-267-54	54	leña	firewood	60	10	FIREWOOD
+Mountain-1978-267-55	55	cielo	sky	61	1732	SKY
+Mountain-1978-267-56	56	lluvia	rain	62	658	RAIN (PRECIPITATION)
+Mountain-1978-267-57	57	viento	wind	63	960	WIND
+Mountain-1978-267-58	58	sol	sun	64	1343	SUN
+Mountain-1978-267-59	59	luna	moon	65	1313	MOON
+Mountain-1978-267-60	60	estrella	star	66	1430	STAR
+Mountain-1978-267-61	61	día	day	67	1225	DAY (NOT NIGHT)
+Mountain-1978-267-62	62	noche	night	68	1233	NIGHT
+Mountain-1978-267-63	63	trueno	thunder	69	1150	THUNDER
+Mountain-1978-267-64	64	relámpago	lightning	70	1654	LIGHTNING
+Mountain-1978-267-65	65	arco iris	rainbow	71	1733	RAINBOW
+Mountain-1978-267-66	66	tierra	earth	72	1228	EARTH (SOIL)
+Mountain-1978-267-67	67	piedra	stone	73	857	STONE
+Mountain-1978-267-68	68	arena	sand	74	671	SAND
+Mountain-1978-267-69	69	casa	house	75	1252	HOUSE
+Mountain-1978-267-70	70	techo	roof	76	769	ROOF
+Mountain-1978-267-71	71	puerta	doorway	77	1567	DOOR
+Mountain-1978-267-72	72	banco	seat, stool	78	2058	STOOL
+Mountain-1978-267-73	73	hamaca	hammock	80	2019	HAMMOCK
+Mountain-1978-267-74	74	catre, cama de barbacoa	bed	81	1663	BED
+Mountain-1978-267-75	75	olla	cooking pot	82	3653	COOKING POT
+Mountain-1978-267-76	76	caserio	village	84	930	VILLAGE
+Mountain-1978-267-77	77	camino, sendero, trocha	path, trail	85	2457	PATH OR ROAD
+Mountain-1978-267-78	78	red	fish net	86	388	FISHNET
+Mountain-1978-267-79	79	anzuelo	fish hook	87	1074	FISHHOOK
+Mountain-1978-267-80	80	hacha	axe	88	677	AXE
+Mountain-1978-267-81	81	cuchillo	knife	89	1352	KNIFE
+Mountain-1978-267-82	82	canoa	canoe	90	1970	CANOE
+Mountain-1978-267-83	83	remo	paddle	91	234	PADDLE
+Mountain-1978-267-84	84	lanza	spear	93	945	SPEAR
+Mountain-1978-267-85	85	arco	bow	94	994	BOW
+Mountain-1978-267-86	86	fiecha	arrow	95	977	ARROW
+Mountain-1978-267-87	87	cerbatana	blowgun	96	432	BLOWGUN
+Mountain-1978-267-88	88	danta	tapir	97	312	TAPIR
+Mountain-1978-267-89	89	tigre	jaguar	98		
```

Comment:
1250 JAGUAR? (tigre tends to refer to the local big cat in Spanish as far as I know, so perhaps we can rely on the English here)